content/docs/command-reference/init.md

```diff
@@ -32,67 +32,59 @@ that are hidden from the user. This directory is automatically staged with
 
 `--subdir` must be provided to initialize DVC in a subdirectory of a Git
 repository. DVC still expects to find the Git repository (will check all
-directories up to the root to find `.git`). This options does not affect any
-config files, `.dvc` directory is created the same way as in the default mode.
-This way multiple DVC projects (including nested ones) could be initialized in a
-single Git repository providing isolation and granular project management.
+directories up to the system root to find `.git/`). This options does not affect
+any config files, `.dvc/` directory is created the same way as in the default
+mode. This way multiple <abbr>DVC projects</abbr> can be initialized in a single
+Git repository, providing isolation between projects.
 
 #### When is this useful?
 
-`--subdir` is mostly used in the scenario of a
-[monorepo](https://en.wikipedia.org/wiki/Monorepo), but also can be used in
-other workflows when such isolation and/or advanced granularity is needed.
+This option is mostly useful in the scenario of a
+[monorepo](https://en.wikipedia.org/wiki/Monorepo) (Git repository split into
+several project directories), but can also be used with other patterns when such
+isolation is needed. `dvc init --subdir` mitigates the issues of initializing
```

Comment:
same here - what is the reason in removing granularity?

Reply:
Simplifying the language. "isolation" is enough?

Reply:
hard to tell ... I guess my thought process was something around trying to include both terms since they resonate with different people (easier to search). `isolation` (implies granularity, but not the other way around) is def enough but a bit too technical, while `granularity` can be a bit more problem related. 

anyways, it's a minor thing - I'm fine if we omit this - your call.

Reply:
I see. But I just think "isolation" is a much more common term: https://trends.google.com/trends/explore?date=today%205-y&q=isolation,granularity

Reply:
They are not the same, so doesn't make sense to my mind to compare them directly. Isolation probably implies granularity, granularity does not necessarily mean isolation.

Reply:
I just don't see what granularity is provided by --subdir. TBH I don't even know what we meant by "granular" here.

![image](https://user-images.githubusercontent.com/1477535/89368347-edbe3b00-d6a0-11ea-98be-2b194a64f5ee.png)

Definitely isolation, though.

Reply:
simple example:

granularity - you split project in a way that `dvc pull` becomes _granular_ (on a project level) because of _isolation_